### DISCUSSION TITLE
Help with Modding

### DISCUSSION THREAD

**Discussion by Dog517** [FabricMC/fabric-api#2418] [Mod Dev Support] [answered]
So, I was adding stuff to a mod I made, and when I went to test it, all the textures in the world were broken. (entitys still rendered right).
here is the Github repo: https://github.com/Dog517/fabric-StuffMod-mod-1.18

**Comment by warjort** [comment, marked as answer]
You can't just post a link a to your mod and say "please fix". :-)

You need to determine which part of your mod is causing the problem.
Then if you are struggling to understand how to fix it yourself you can ask here.

**Reply by Dog517** [reply]
Yeah, I already fixed this. I just had to resize the textures to be powers of 2.
